Sources/PointFreeTestSupport/TestCase.swift

```diff
@@ -3,47 +3,29 @@ import Models
 import NIO
 @testable import PointFree
 import PointFreeRouter
+import PostgresKit
 import Prelude
 import SnapshotTesting
 import XCTest
 
 open class LiveDatabaseTestCase: TestCase {
+  var database: Database.Client!
+  var pool: EventLoopGroupConnectionPool<PostgresConnectionSource>!
+  let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+
   override open func setUp() {
     super.setUp()
 
-    Current.database = .liveTest
-
-    _ = try! Current.database.execute("DROP SCHEMA IF EXISTS public CASCADE", [])
-      .flatMap(const(Current.database.execute("CREATE SCHEMA public", [])))
-      .flatMap(const(Current.database.execute("GRANT ALL ON SCHEMA public TO pointfreeco", [])))
-      .flatMap(const(Current.database.execute("GRANT ALL ON SCHEMA public TO public", [])))
-      .flatMap(const(Current.database.migrate()))
-      .flatMap(const(Current.database.execute("CREATE SEQUENCE test_uuids", [])))
-      .flatMap(const(Current.database.execute("CREATE SEQUENCE test_shortids", [])))
-      .flatMap(const(Current.database.execute(
-        """
-          CREATE OR REPLACE FUNCTION uuid_generate_v1mc() RETURNS uuid AS $$
-          BEGIN
-            RETURN ('00000000-0000-0000-0000-'||LPAD(nextval('test_uuids')::text, 12, '0'))::uuid;
-          END; $$
-          LANGUAGE PLPGSQL;
-          """,
-        []
-      )))
-      .flatMap(const(Current.database.execute(
-        """
-          CREATE OR REPLACE FUNCTION gen_shortid(table_name text, column_name text)
-          RETURNS text AS $$
-          BEGIN
-            RETURN table_name||'-'||column_name||nextval('test_shortids')::text;
-          END; $$
-          LANGUAGE PLPGSQL;
-          """,
-        []
-      )))
-      .run
-      .perform()
-      .unwrap()
+    self.pool = EventLoopGroupConnectionPool(
+      source: PostgresConnectionSource(
+        configuration: PostgresConfiguration(
+          url: "postgres://pointfreeco:@localhost:5432/pointfreeco_test"
```

Comment:
should we use Current.mock.envVars to get this url? and maybe put an assertion at the end of `LiveDatabaseTest` to make sure we aren't connecting to a heroku db?